frontend/package.json

```diff
@@ -4,10 +4,10 @@
     "node": ">=0.10.3 <17"
   },
   "scripts": {
-    "start": "expo start",
+    "start": "nvm use 16 && expo start",
     "android": "expo start --android",
     "ios": "expo start --ios",
-    "web": "expo start --web",
+    "web": "nvm use 16 && expo start --web",
     "eject": "expo eject"
   },
   "dependencies": {
```

Comment:
The code patch updates the "start" and "web" scripts in the package.json file to include the command "nvm use 16" before running "expo start" or "expo start --web". This ensures that Node.js version 16 is used for these commands.

Overall, this code patch seems fine, but there are a couple of things to consider:

1. Using "nvm use" inside scripts can be problematic if the user doesn't have NVM installed or hasn't set up a default Node.js version. A more robust approach would be to specify the Node.js version directly in the "engines" field of the package.json file.

2. If the project is being developed on multiple machines with different Node.js versions, it might be better to specify the Node.js version required for the project as a whole, rather than just for the "start" and "web" scripts.